.github/pull_request_template.md

```diff
@@ -9,29 +9,22 @@ It is EXTREMELY common for PRs to be held up due to trivial changes being reques
 <!--
 A one-line description of your change that will be extracted and added to the [project changelog](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/changelog.txt).
 
-The format is (ignore the square brackets): ```SUMMARY: [Category] "[description]"```
+The format is (ignore the square brackets):
 
-The categories to choose from are:
+SUMMARY: [Category] "[description]"
 
-* Features
-* Content
-* Interface
-* Mods
-* Balance
-* Bugfixes
-* Performance
-* Infrastructure
-* Build
-* I18N
+The categories to choose from are: Features, Content, Interface, Mods, Balance, Bugfixes, Performance, Infrastructure, Build, I18N
 
-Example: ```SUMMARY: Content "Adds new mutation category 'Mouse'"```
+Example:
+
+SUMMARY: Content "Adds new mutation category 'Mouse'"
```

Comment:
Would you mind fixing the tense here while you're at it?
```suggestion
SUMMARY: Content "Add new mutation category 'Mouse'"
```

Reply:
Good change, imperative is better. Thanks!